src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -240,19 +232,17 @@ public function __construct($options = [])
 
         $this->scopes = $options['scopes'];
 
-        $createStubOptions = [];
-        if (array_key_exists('sslCreds', $options)) {
-            $createStubOptions['sslCreds'] = $options['sslCreds'];
-        }
-        $this->grpcCredentialsHelper = new GrpcCredentialsHelper($options);
-
-        $createOperationsStubFunction = function ($hostname, $opts, $channel) {
-            return new OperationsGrpcClient($hostname, $opts, $channel);
-        };
-        if (array_key_exists('createOperationsStubFunction', $options)) {
-            $createOperationsStubFunction = $options['createOperationsStubFunction'];
+        if (empty($options['createOperationsStubFunction'])) {
```

Comment:
We should rename this option to `createOperationsTransportFunction`

Reply:
Then we should also rename `$operationsStub` to `$operationsTransport`